docs/examples/audio_decoder.ipynb

```diff
@@ -0,0 +1,251 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import simpleaudio as sa\n",
+    "import numpy as np\n",
+    "\n",
+    "\n",
+    "wav = sa.WaveObject.from_wave_file(\"audio/wav/three.wav\")\n",
+    "three_audio = np.frombuffer(wav.audio_data, dtype=np.int16)\n",
+    "\n",
+    "print(\"src: simpleaudio\")\n",
+    "print(\"shape: \", three_audio.shape)\n",
+    "print(\"data: \", three_audio)\n",
+    "print(\"\\n\")\n",
+    "print(\"src: DALI\")\n",
+    "print(\"shape: \", audio_data[0].shape)\n",
+    "print(\"data: \", audio_data[0])\n",
```

Comment:
You can also compare audio_data with three_audio to check if they matches

Reply:
Done